### PR TITLE
Reword changelog entries

### PR DESCRIPTION
**Pull request by fd0**
Improve the text for the changelog files.

### CODE REVIEW

File: changelog/unreleased/issue-2742 [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
 restic did not detect when no data was returned at all. This could cause
 for example the `check` command to report the following error:
-```
-Pack ID does not match, want xxxxxxxx, got e3b0c442
-```
+
+    Pack ID does not match, want [...], got e3b0c442
 
 This has been fixed by correctly detecting the incomplete download and
 retrying the download.
```
Comment (rawtaz) [inline changelog/unreleased/issue-2742:10]: Suggest "This has been fixed by correctly detecting and retrying the incomplete download".

File: changelog/unreleased/issue-2742 [hunk 2]
```diff
@@ -1,11 +1,10 @@
 Bugfix: Improve error handling for rclone and rest backend over HTTP2
```
Comment (rawtaz) [inline changelog/unreleased/issue-2742:1]: s/rest/REST/

File: changelog/unreleased/issue-3312 [hunk 3]
```diff
@@ -1,6 +1,6 @@
 Enhancement: Add auto-completion file for fish
 
-The version of cobra had been increased to v1.1.3, and `generate` command has added 
-auto-completion file for fish.
+We added support for fish auto completion for the `generate` command by
+upgrading the Cobra library.
```
Comment (rawtaz) [inline changelog/unreleased/issue-3312:4]: No biggie but it's not really needed to explain how a fix was actually implemented (in this case "by upgrading the Cobra library", that's besides the point :D

File: changelog/unreleased/issue-3312 [hunk 4]
```diff
@@ -1,6 +1,6 @@
 Enhancement: Add auto-completion file for fish
```
Comment (rawtaz) [inline changelog/unreleased/issue-3312:1]: s/file/support/

File: changelog/unreleased/issue-3377 [hunk 5]
```diff
@@ -1,6 +1,6 @@
 Enhancement: Add release binaries for Apple Silicon
 
-We've added release binaries for macOS on Apple Silicon.
+We've added release binaries for macOS on Apple Silicon (m1).
```
Comment (rawtaz) [inline changelog/unreleased/issue-3377:3]: s/m1/M1/

File: changelog/unreleased/issue-3414 [hunk 6]
```diff
@@ -1,19 +1,19 @@
-Enhancement: Add --keep-within-hourly switch to restic forget
+Enhancement: Add `--keep-within-hourly` option to restic forget
 
-restic forget allowed users to specify keeping a given number of 
+Restic forget allowed users to specify keeping a given number of
 hourly backups, or to keep all backups within a given interval,
 but not both, that is to specify keeping hourly backups within
 a given interval.
```
Comment (rawtaz) [inline changelog/unreleased/issue-3414:6]: Suggest "The `forget` command allowed keeping a given number of hourly backups or to keep all backups within a given interval, but it was not possible to specify keeping hourly backups within a given interval.".

File: changelog/unreleased/issue-3414 [hunk 7]
```diff
@@ -1,19 +1,19 @@
...
 
-The --keep-within-hourly switch offers this functionality, and 
-parallell switches for daily/weekly/monthly/yearly are also 
-implemneted. The new switches are:
+The `--keep-within-hourly` option offers this functionality and
+similar options for daily/weekly/monthly/yearly are also
+implemented. The new flags are:
```
Comment (rawtaz) [inline changelog/unreleased/issue-3414:10]: Suggest "The new `--keep-within-hourly` option now offer this functionality. Similar options for daily/weekly/monthly/yearly are also implemented - the new options are:"

File: changelog/unreleased/issue-3247 [hunk 8]
```diff
@@ -1,8 +1,8 @@
 Change: Empty files now have size of 0 in restic ls --json output
 
-Restic ls --json used to omit the sizes of empty files in its output. It now
-reports "size":0 explicitly for regular files, while omitting the size field
-for all other types.
+Restic's `ls --json` command used to omit the sizes of empty files in its
```
Comment (rawtaz) [inline changelog/unreleased/issue-3247:3]: Suggest starting with "The `ls --json` command ..".

File: changelog/unreleased/issue-3247 [hunk 9]
```diff
@@ -1,8 +1,8 @@
 Change: Empty files now have size of 0 in restic ls --json output
```
Comment (rawtaz) [inline changelog/unreleased/issue-3247:1]: Suggest "Change: Empty files now have size of 0 in `ls --json` output" (the restic context is a given).

File: changelog/unreleased/pull-3167 [hunk 10]
```diff
@@ -4,6 +4,6 @@ The `--last` option allowed limiting the output of the `snapshots`
 command to the latest snapshot for each host. The new `--latest n`
 option allows limiting the output to the latest `n` snapshots.
 
-This change deprecate `--last` in favour of `--latest 1`
+This change deprecates the option `--last` in favour of `--latest 1`.
 
 https://github.com/restic/restic/pull/3167
```
Comment (rawtaz) [inline changelog/unreleased/pull-3167:9]: Couldn't add a comment to the top-most line. Suggest changing heading to "Enhancement: Allow specifying limit of `snapshots` list".

File: changelog/unreleased/pull-3427 [hunk 11]
```diff
@@ -1,10 +1,10 @@
 Enhancement: `find --pack` fallback to index if data file is missing
 
-When investigating a repository with missing data files, it might be useful
-to determine affected snapshots before running `rebuild-index`.
-`restic find --pack pack-id` previously returned no data as it required
-accessing the data file. Now, if the necessary data is still available in the
-repository index, it gets retrieved from there.
+When investigating a repository with missing data files, it might be useful to
+determine affected snapshots before running `rebuild-index`. Previously,
+`restic find --pack pack-id` returned no data as it required accessing the data
```
Comment (rawtaz) [inline changelog/unreleased/pull-3427:5]: Suggest removing "restic" before "find" as the context is given.

**Comment by rawtaz** [comment]
I added one commit with some polishing, feel free to squash it or just pick what you want.

**Review by rawtaz — APPROVED** [review]

**Comment by fd0** [comment]
Thanks!

**Pull request merged**